apps/communications/contacts/test/marionette/import_sdcard_test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+var Contacts = require('./lib/contacts');
+var assert = require('assert');
+
+marionette('Import from SDCARD', function() {
+  var profile = Contacts.config;
+  // Enable device storage
+  var prefs = ['device.storage.enabled',
+        'device.storage.testing',
+        'device.storage.prompt.testing'];
+  prefs.forEach(pref => {
+    profile[pref] = true;
+  });
+  var client = marionette.client({
+    profile: profile,
+    desiredCapabilities: { raisesAccessibilityExceptions: false }
+  });
+  var actions, subject, selectors;
+
+  setup(function() {
+    // Remove all files in temp device storage.
+    client.fileManager.removeAllFiles();
+
+    // Add the default vcard file
+    client.fileManager.add({
+      type: 'text',
+      filePath: 'apps/communications/contacts/test/marionette/data/vcard_4.vcf'
+    });
+
+    subject = new Contacts(client);
```

Comment:
TL;DR: I'd use `contactsApp` instead of `subject`.

I know it's been used at some other places in the tests, and I'm fine if we don't change it.

However, `subject` is not a self-explanatory name. You have to do an indirection (that is to say, either remember, or scroll back where it's defined, or take a look at the file path) to remember that's the contact app you're testing. This can be confusing if you read many tests at the same time. For example, if somebody breaks many tests in many apps, because he/she changed something in `/shared`.